CMake errors out due to re_enable_warnings.hpp being removed

Since a7a7f1ed9f27e1ea111c364f0d3f41763e8356f3 CMake reports an error:
```
CMake Error at CMakeLists.txt:72 (list):
  list sub-command REMOVE_ITEM requires two or more arguments.
Call Stack (most recent call first):
  CMakeLists.txt:79 (range_v3_list_remove_glob)
```

Cause: include/range/v3/detail/re_enable_warnings.hpp is not present anymore.

Tested on a fresh build with CMake 3.0.2 and 3.5.2. I am not sure why Travis CI builds are happy with this.

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -76,9 +76,6 @@ endmacro()
 if(NOT RANGE_V3_NO_HEADER_CHECK)
   include(CheckIncludeFileCXX)
   file(GLOB_RECURSE RANGE_V3_PUBLIC_HEADERS "${CMAKE_SOURCE_DIR}/include/*.hpp")
-  range_v3_list_remove_glob(RANGE_V3_PUBLIC_HEADERS GLOB_RECURSE
-    "${CMAKE_SOURCE_DIR}/include/range/v3/detail/re_enable_warnings.hpp"
-    )
 
   foreach(_header IN LISTS RANGE_V3_PUBLIC_HEADERS)
     file(RELATIVE_PATH _relative "${CMAKE_SOURCE_DIR}/include" "${_header}")

